config/vite.lib.config.js

```diff
@@ -18,13 +18,7 @@ export default defineConfig({
     tsconfigPaths(),
     dts({
       exclude: ['**/*.spec.ts', '**/*.test.ts', '**/tests/**'],
-      afterBuild: () => {
-        const types = fs.readdirSync(path.join(PWD, 'dist/src'))
-        types.forEach((file) => {
-          fs.renameSync(path.join(PWD, 'dist/src', file), path.join(PWD, 'dist', file))
-        })
-        fs.rmdirSync(path.join(PWD, 'dist/src'))
-      }
+      entryRoot: 'src'
```

Comment:
new `entryRoot` param in the latest dts plugin version that replaces our `afterBuild` script

Reply:
Much cleaner. I wish I saw this prop when creating the config 🤦🏼